### PR TITLE
Merge test to master

### PR DESCRIPTION
**Pull request by lunchmap**

### CODE REVIEW

**Comment by lunchmap** [comment]
TEST

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<head>
+</head>
+<body>
+    <script type='text/javascript'>
+    <!--
+        location.href='https://github.com/lunchmap/lunchmap.github.io'
```
Comment (Ohmry) [inline index.html]: test

File: index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    //-->
+    </script>
+</body>
+</html>
```
Comment (lunchmap) [inline index.html]: OK Done.

File: index.html (hunk 1 above)
Comment (lunchmap) [inline index.html]: 이렇게 하는건가요?

Comment (Ohmry) [inline index.html]: 핳..

File: index.html (hunk 2 above)
Comment (Ohmry) [inline index.html]: Thanks.

Comment (Ohmry) [inline index.html]: Try review from client

**Pull request merged**